### PR TITLE
docs: fix mistakes in the English version of cloud docs

### PR DESCRIPTION
**Pull request by ShuaoZhang**
Signed-off-by: shuaoz <shuaoz@umich.edu>

### CODE REVIEW

File: docs/4.0/docs/cloud/apps/scp/scp.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ Download the user-specific kubeconfig file from the sealos cloud:
 
 ![img_4.png](img_4.png)
 
-Copy to the .kube/config file, e.g. `cp kubeconfig.yaml .kube/config`
+Copy it to the .kube/config folder, e.g. `cp kubeconfig.yaml .kube/config`
```
Comment (zzjin) [inline docs/4.0/docs/cloud/apps/scp/scp.md:105]: `.kube/config` is the kubeconfig file, not the folder stores it. So just keep original and do not changed to `folder`.

File: docs/4.0/docs/cloud/Intro.md [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
 cloud-native (built around the technical ecosystem of kubernetes) architecture and 100% opensource.
 
 ## why sealos Cloud
 
-* The product is simple and clean, if you have been tortured by the dazzling advertising discount information of other Cloud provider.
-* Completely open source, which means unlimited and free to customize.
-* The public cloud and private cloud is exactly the same. The public cloud and private cloud of sealos have the same set of codes. 
-* Runs everywhere, compatible with various other cloud infrastructures, compatible with bare metal, supports various architectures and mainstream linux operating systems.
-* The price is cheap, and privatization can help enterprises save 80% of infrastructure costs, and direct use of public clouds can save 10% to 40% of costs.
+* The product is simple and clean. You won't be tortured by the dazzling advertisements published by other Cloud providers.
```
Comment (zzjin) [inline docs/4.0/docs/cloud/Intro.md:15]: Here, we means "dazzling advertising discount information", but not "azzling advertisements".

**Review by zzjin — CHANGES_REQUESTED** [review]
Good pr! some docs is miss-understanding, and when fixed, we can merge it asap.

File: docs/4.0/docs/cloud/apps/scp/scp.md (hunk 1 above)
Comment (ShuaoZhang) [reply docs/4.0/docs/cloud/apps/scp/scp.md]: Modified.

File: docs/4.0/docs/cloud/Intro.md (hunk 2 above)
Comment (ShuaoZhang) [reply docs/4.0/docs/cloud/Intro.md]: Done

**Review by zzjin — APPROVED** [review]

**Pull request merged**